docs/developer/decision-records/2022-04-21-dpf-blob-data-transfer/README.md

```diff
@@ -0,0 +1,38 @@
+# Blob storage transfer
+
+ADR describing the blob storage transfer end to end flow between 2 participants.
+
+## Description
+
+The data-plane-azure-storage extension can be used on DPF to support blob transfers.
+A client can trigger a blob transfer on the consumer side via the Data Management API.
+The client needs to use the managedResources=true option in its HTTP request, otherwise the transfer will not work. Client would not be aware that an error occurred, it would just never see the transferProcess with completed state when polling for the result.   
+Storage accounts access key should be stored in advanced in Keyvaults. The consumer can generate SAS token to give the provider the possibility to write data to its container.
+
+## Sequence diagram
+
+The following sequence diagram describes the flow to transfer a blob from a Provider storage account to a consumer storage account.
+It starts from the client triggering the transfer on the consumer side and finishes when the consumer deletes the blob
+after the client triggered the data deletion.
+
+![blob-transfer](../../../diagrams/blob-transfer.png)
+
+1. The client calls the data management API to trigger a transfer process. managedResources needs to be set to true, to make sure that the consumer provisions the blob container. If it is not set to true, the transfer will not work. Then, the transfer would never be completed. 
```

Comment:
What is the use case for `POST /transferprocess` when `managedResources` is set to `false`? It would help to understand why it needs set to `true`.

Reply:
Makes sense thanks, I added an explanation.